feat: document users.ban and users.offboard endpoints (#41)
ulule/developers.ulule.com: content/reference/user.md

| --------- | ------ | ----------------------------------------------------------------------- |
| `message` | string | Body of the first message in the thread                                 |
| `subject` | string | Subject of the thread -- required, must be between 3 and 255 characters |

## Ban user

Ban the user with the given id.

> This endpoint is only accessible by staff.

Reviewer: ```suggestion
This endpoint is only accessible to staff.
```

{{% http method="post" %}}/v1/users/:id/ban{{% /http %}}

| Parameter | Description  |

> | --------- | ------------ |

Reviewer: format the table

| `:id`     | User ID      |

## Offboard team member

Offboard the user with the given id.

> This endpoint is only accessible by staff.

Reviewer: ```suggestion
This endpoint is only accessible to staff.
```

> If another ID is given in the payload the projects and related

Reviewer: ```suggestion
If `user_id` is present in the payload, it substitutes the offboarded user.
```

discussions managed by the offboarded user are attached to the user with the ID from the payload.

{{% http method="post" %}}/v1/users/:id/offboard{{% /http %}}

| Parameter | Description  |

> | --------- | ------------ |

Reviewer: format the table

| `:id`     | User ID      |

### Payload

| Field     | Type | Description                                                 |
| --------- | ---- | ----------------------------------------------------------- |

> | `user_id` | int  | User ID for the substitution of project manager -- optional |

Reviewer: ```suggestion
| `user_id` | int  | User ID used to substitute the offboarded user -- optional |
```
